paper.md

```diff
@@ -72,8 +72,9 @@ bibliography: paper.bib
 ---
 # Summary
 
-In recent years the Robotics Operating System [@quigley2009ros] (ROS) has become the 'de facto' standard framework for the development of software in robotics. The idea of `ros_control` originates from the PR2 robot's `pr2_controller_manager` framework with some ideas borrowed from OROCOS [@bruyninckx2001open]. The `ros_control` framework provides the capability to implement and manage robot controllers with a focus on both _real-time performance_ and _sharing of controllers_ in a robot-agnostic way. 
+  In recent years the Robotics Operating System [@quigley2009ros] (ROS) has become the 'de facto' standard framework for the development of software in robotics. The idea of `ros_control` originates from the `pr2_controller_manager` framework specific to the PR2 robot. The primary motivation for a separate framework for robot-control was necessary because ROS doesn't provide a realtime-safe communication layer but the framework implements mechanisms for controller-lifecycle- and hardware resource-  management as well as abstractions on hardware interfaces. The `ros_control` framework provides the capability to implement and manage robot controllers with a focus on both _real-time performance_ and _sharing of controllers_ in a robot-agnostic way.
```

Comment:
the difference with pr2_controller_manager in the generality of the framework. 
ros-control is not binded to any specific robot.

Reply:
I would start by talking about the core idea of ros_control, and after compare it to pr2_controller_manager:
The idea of `ros_control` is to bring real time control capabilities into ROS (which does not have) while keeping the other cool ROS' features such as... . The idea originated from ... .

Reply:
check now